app/src/main/java/com/khmelenko/lab/travisclient/mvp/MvpActivity.java

```diff
@@ -18,6 +19,10 @@
  */
 public abstract class MvpActivity<T extends MvpPresenter> extends AppCompatActivity {
 
+    static {
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
+    }
```

Comment:
That's a workaround for vector images to work inside views that normally don't support it. In this case it's about the circular image view. You can read all the details in here “AppCompat — Age of the vectors” @chrisbanes https://medium.com/@chrisbanes/appcompat-v23-2-age-of-the-vectors-91cbafa87c88